packages/opentelemetry-plugin-express/README.md

```diff
@@ -45,6 +45,10 @@ const provider = new NodeTracerProvider();
 
 See [examples/express](https://github.com/open-telemetry/opentelemetry-js/tree/master/examples/express) for a short example.
 
+### Caveats
+
+Because of the way express works, it's hard to correctly compute the time took that each span took to complete. For this reason the time you'll see reported for each span will **not** represent the time it really took.
```

Comment:
Only for asynchronous spans

Reply:
I don't why but at some point i though that it wasn't possible for sync callback too so i'd removed the acurate reporting for all spans, but you are right. I've repushed the previous implementation that report accurate duration for sync middlewares